src/pages/Committees.js

```diff
@@ -580,11 +634,213 @@ function SportsCommittee() {
+                <li>Email: festmasteriet@medieteknik.com</li>
+                <li>Instagram: @festmasteriet_medieteknik</li>
+            </ul>
+        </p>
+    )
+}
+
+function FilmCommittee () {
+    return (
+        <p>
+            Interested in films? The Film Committee (Swedish: Filmnämden) produces a high-effort film project for each reception.
+            They handle every part of film-making, from screen-writing, to filming, post-production, etc. You can see
+            their work on their social networks below.
+            <br/>
+            Contact:
+            <ul>
+                <li>Email: filmnamnden@medieteknik.com</li>
+                <li>Instagram: @filmnamnden</li>
+                <li><a href="https://youtube.com/@filmnamndenmedieteknik?si=MhexLexOo8W1N8eO">Youtube</a></li>
+            </ul>
+        </p>
+    )
+}
+
+function PhotoGroup () {
+    return (
+        <p>
+            Do you have an interest in photography? The Photo Group (Swedish: Fotogruppen) is a meeting place for all photography people of
+            the Media chapter. You do not need to have technical knowledge of cameras, you can join even if you are only interested in looking
+            at others' pictures. Some members in the committee have extensive technical knowledge, some are more on the art aspect, so
+            there is much you can learn from them!
+            <br/>
+            Contact:
+            <ul>
+                <li>Email: fotogruppen@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function MediaSportsCommittee () {
+    return (
+        <p>
+            Come combine your studies with fresh air, exercising and meeting new friends. The Sports Committee (Swedish: Idrottsnämnden) organizes
+            sport activities for all Media students to enjoy.
+            <br/>
+            Contact:
+            <ul>
+                <li>Email: idrottsnamnden@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function Matlaget () {
+    return (
+        <p>
+            Prepare your apron and best cooking ustensils, it's time to cook. The Food Team (Swedish: Matlaget) prepares the food
+            for Media's chapter meetings, from planning what to cook, to buying the ingredients and finally cooking. The committee is open, i.e. anyone
+            in the Media chapter can join the Food Team and cook as much (or as little) as they want. It's also a great oppportunity to learn cooking, or
+            just to have fun!
+            <br/>
+            Contact:
+            <ul>
+                <li>Email: masterkock@medieteknik.com</li>
+            </ul>
+        </p>
+    )
+}
+
+function MediaMärkt () {
+    return (
+        <p>
+            Your tail-coat (Swedish: B-Frak) feels empty? Media Label (Swedish: Media Märkt) designs and sells patches so that no spot on your tail-coat feels
+            like it misses something. You can also talk to them if you have questions about overalls.
```

Comment:
```suggestion
            Does your tail-coat (Swedish: B-Frak) feel empty? Media Label (Swedish: Media Märkt) designs and sells patches so that no spot on your tail-coat feels
            like it's missing something. You can also talk to them if you have questions about overalls.
```